%\VignetteEngine{knitr::knitr}
%\VignetteIndexEntry{Single cell RNA seq data due to Jaitin et al (2014), Deng et al (2014) and Zeisel et al (2015)}
%\VignettePackage{singlecellRNAseqData}

% To compile this document
% library('knitr'); rm(list=ls()); knit('singlecellRNAseqData/vignettes/data-vignette.Rnw')
% library('knitr'); rm(list=ls()); knit2pdf('singlecellRNAseqData/vignettes/data-vignette.Rnw'); openPDF('data-vignette.pdf')
%

\documentclass[12pt]{article}

\newcommand{\singlecellRNAseqData}{\textit{singlecellRNAseqData}}
\usepackage{dsfont}
\usepackage{cite}




\RequirePackage{/Library/Frameworks/R.framework/Versions/3.3/Resources/library/BiocStyle/resources/tex/Bioconductor}

\AtBeginDocument{\bibliographystyle{/Library/Frameworks/R.framework/Versions/3.3/Resources/library/BiocStyle/resources/tex/unsrturl}}



\author{Kushal K Dey, Chiaowen Joyce Hsiao \& Matthew Stephens \\[1em]
\small{\textit{Stephens Lab}, The University of Chicago} \mbox{ }\\
\small{\texttt{$^*$Correspondending Email: mstephens@uchicago.edu}}}

\bioctitle[Single-cell RNA-Seq data sets]{Single-cell RNA-Seq data sets}

\begin{document}

\maketitle

\begin{abstract}
\vspace{1em}
 This package includes three single-cell RNA-seq data sets: 1) Mouse spleen cells due to Jaitin et al 2014 \cite{Jaitin2014}, 2) Mouse preimplantation embryos due to Deng et al
 2014 \cite{Deng2014}, and 3) Mouse cortex and hippocampus cells data due to
 Zeisel et al 2015 \cite{Zeisel2015}. Each data set is stored in an \textit{ExpressionSet} objects. We previously analyzed Jaitin et al and Deng et al in the manuscript entitled "Clustering RNA-seq expression data using grade of membership models", Kushal K Dey, Chiaowen Joyce Hsiao, and Matthew Stephens (2016).

\vspace{1em}
\textbf{\singlecellRNAseqData{} version:} 0.99.0 \footnote{This document used the vignette from \Bioconductor{} package \Biocpkg{DESeq2, cellTree, CountClust} as \CRANpkg{knitr} template}
\end{abstract}




\newpage

\tableofcontents

\section{Installation}

To install the Bioconductor version of this package,

\begin{knitrout}
\definecolor{shadecolor}{rgb}{0.969, 0.969, 0.969}\color{fgcolor}\begin{kframe}
\begin{alltt}
\hlkwd{source}\hlstd{(}\hlstr{"http://bioconductor.org/biocLite.R"}\hlstd{)}
\hlkwd{biocLite}\hlstd{(}\hlstr{"singlecellRNAseqData"}\hlstd{)}
\end{alltt}
\end{kframe}
\end{knitrout}

To install the working version from Github, the user needs CRAN package
\CRANpkg{devtools}.

\begin{knitrout}
\definecolor{shadecolor}{rgb}{0.969, 0.969, 0.969}\color{fgcolor}\begin{kframe}
\begin{alltt}
\hlkwd{library}\hlstd{(devtools)}
\hlkwd{install_github}\hlstd{(}\hlstr{"kkdey/singlecellRNAseqData"}\hlstd{)}
\end{alltt}
\end{kframe}
\end{knitrout}

To load the package

\begin{knitrout}
\definecolor{shadecolor}{rgb}{0.969, 0.969, 0.969}\color{fgcolor}\begin{kframe}
\begin{alltt}
\hlkwd{library}\hlstd{(singlecellRNAseqData)}
\end{alltt}
\end{kframe}
\end{knitrout}

We now provide a brief summary of the three datasets hosted in this package and
how the user can extract different features of the data from the
\textit{ExpressionSet} framework in which the data is stored.

\section{Deng et al (2014)}

Deng et al (2014) \cite{Deng2014} collected single-cell RNA-seq expression data from embryos across mouse embryonic preimplantation developmental stages (zygote, 2 cell,
4 cell, 8 cell, 16 cell,early blastocyst, mid blastocyst and late blastocyst
stages). The data in this package includes read counts of $22,431$ genes (after removing SmartSeq2 and pooled samples) from $259$ single cells. We downloaded the data from Gene Expression Omnibus
(GEO:GSE45719: see \url{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE45719})

























