%\VignetteEngine{knitr::knitr}
%\VignetteIndexEntry{Single cell RNA seq data due to Jaitin et al (2014), Deng et al (2014) and Zeisel et al (2015)}
%\VignettePackage{singlecellRNAseqData}

% To compile this document
% library('knitr'); rm(list=ls()); knit('singlecellRNAseqData/vignettes/data-vignette.Rnw')
% library('knitr'); rm(list=ls()); knit2pdf('singlecellRNAseqData/vignettes/data-vignette.Rnw'); openPDF('data-vignette.pdf')
%

\documentclass[12pt]{article}

\newcommand{\singlecellRNAseqData}{\textit{singlecellRNAseqData}}
\usepackage{dsfont}
\usepackage{cite}

<<knitr, echo=FALSE, results="hide">>=
library("knitr")
opts_chunk$set(tidy=FALSE,tidy.opts=list(width.cutoff=30),dev="png",
               fig.show="hide",fig.width=4,fig.height=7,
               message=FALSE)
@


<<style, eval=TRUE, echo=FALSE, results="asis">>=
BiocStyle::latex()
@


\author{Kushal K Dey, Chiaowen Joyce Hsiao \& Matthew Stephens \\[1em]
\small{\textit{Stephens Lab}, The University of Chicago} \mbox{ }\\
\small{\texttt{$^*$Correspondending Email: mstephens@uchicago.edu}}}

\bioctitle[Single-cell RNA-Seq data sets]{Single-cell RNA-Seq data sets}

\begin{document}

\maketitle

\begin{abstract}
\vspace{1em}
 This package includes three single-cell RNA-seq data sets: 1) Mouse spleen cells due to Jaitin et al 2014 \cite{Jaitin2014}, 2) Mouse preimplantation embryos due to Deng et al
 2014 \cite{Deng2014}, and 3) Mouse cortex and hippocampus cells data due to
 Zeisel et al 2015 \cite{Zeisel2015}. Each data set is stored in an \textit{ExpressionSet} objects. We previously analyzed Jaitin et al and Deng et al in the manuscript entitled "Clustering RNA-seq expression data using grade of membership models", Kushal K Dey, Chiaowen Joyce Hsiao, and Matthew Stephens (2016).

\vspace{1em}
\textbf{\singlecellRNAseqData{} version:} \Sexpr{packageDescription("singlecellRNAseqData")$Version} \footnote{This document used the vignette from \Bioconductor{} package \Biocpkg{DESeq2, cellTree, CountClust} as \CRANpkg{knitr} template}
\end{abstract}


<<options, results="hide", echo=FALSE>>=
options(digits=3, width=80, prompt=" ", continue=" ")
@

\newpage

\tableofcontents

\section{Installation}

To install the Bioconductor version of this package,

<<install_bioc, echo=TRUE, eval=FALSE>>=
source("http://bioconductor.org/biocLite.R")
biocLite("singlecellRNAseqData")
@ %def

To install the working version from Github, the user needs CRAN package
\CRANpkg{devtools}.

<<install_github, echo=TRUE, eval=FALSE>>=
library(devtools)
install_github("kkdey/singlecellRNAseqData")
@ %def

To load the package

<<load, echo=TRUE, eval=TRUE>>=
library(singlecellRNAseqData)
@

We now provide a brief summary of the three datasets hosted in this package and
how the user can extract different features of the data from the
\textit{ExpressionSet} framework in which the data is stored.

\section{Deng et al (2014)}

Deng et al (2014) \cite{Deng2014} collected single-cell RNA-seq expression data from embryos across mouse embryonic preimplantation developmental stages (zygote, 2 cell,
4 cell, 8 cell, 16 cell,early blastocyst, mid blastocyst and late blastocyst
stages). The data in this package includes read counts of $22,431$ genes (after removing SmartSeq2 and pooled samples) from $259$ single cells. We downloaded the data from Gene Expression Omnibus
(GEO:GSE45719: see \url{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE45719})

<<deng_data, echo=TRUE, eval=TRUE>>=
data("MouseDengESC")
MouseDengESC
@ %def


Expression data for the first few genes (along rows) and the first few
cells in the sample (along columns):

<<deng_exprs, echo=TRUE, eval=TRUE>>=
counts_deng <- Biobase::exprs(MouseDengESC)
head(counts_deng[,1:5])
@ %def


Phenotype or metadata sample information are stored in the `phenoData` slot and include both the development stage of each cell and the corresponding embryo ID. The development
stage of each cell can be extracted as follows:

<<deng_pdata, echo=TRUE, eval=TRUE>>=
pheno_deng <- Biobase::pData(MouseDengESC)
colnames(pheno_deng)
table(pheno_deng$cell_type)
@ %def


Gene information are stored in `featureData` slot:

<<deng_features, echo=TRUE, eval=TRUE>>=
genes_deng <- Biobase::fData(MouseDengESC)
head(genes_deng)
@ %def


\section{Jaitin et al (2014)}

Jaitin et al (2014) \cite{Jaitin2014} collected single cell RNAseq data of mouse spleen cells using Unique Molecule Identifiers (UMIs). $4,590$ single cells were included with $20,190$ genes. The dataset includes in this package was downloaded from Gene Expression Omnibus (GEO:GSE54006: see
\url{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54006})


<<jaitin_data, echo=TRUE, eval=TRUE>>=
data("MouseJaitinSpleen")
MouseJaitinSpleen
@ %def


Expression data for the first few genes (along rows) and the first few
cells in the sample (along columns):

<<jaitin_exprs, echo=TRUE, eval=TRUE>>=
counts_jaitin <- Biobase::exprs(MouseJaitinSpleen)
head(counts_jaitin[,1:5])
@ %def


Phenotype or metadata of the samples include mouse ID, well ID, amplification batch, sequencing batch and can be extracted as follows:

<<jaitin_pdata, echo=TRUE, eval=TRUE>>=
pheno_jaitin <- Biobase::pData(MouseJaitinSpleen)
head(pheno_jaitin[ ,c("amplification_batch",
                      "sorting_markers",
                      "well_id",
                      "ERCC_dilution")])
@ %def


Gene names are stored in the `featureData` slot:

<<jaitin_features, echo=TRUE, eval=TRUE>>=
genes_jaitin <- Biobase::featureNames(MouseJaitinSpleen)
head(genes_jaitin)
@ %def


\section{Zeisel et al (2015)}

Zeisel et al (2015) \cite{Zeisel2015} collected single cell RNA-seq data from Mouse
cortex and hippocampus, with the goal of identifying different cell types. Read counts of $19,968$ genes were included in the released data set for $3,005$ single cells. The data in this package was downloaed from Gene Expression Omnibus (GEO:GSE60361: see
\url{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60361})


<<zeisel_data, echo=TRUE, eval=TRUE>>=
data("MouseZeiselBrain")
MouseZeiselBrain
@ %def


Expression data for the first few genes (along rows) and the first few
cells in the sample (along columns):

<<zeisel_exprs, echo=TRUE, eval=TRUE>>=
counts_zeisel <- Biobase::exprs(MouseZeiselBrain)
head(counts_zeisel[,1:5])
@ %def


Phenotype or metadata of the samples include tissue of origin, class type of cells, age and sex of subjects from whom the cells were extracted.

<<zeisel_pdata, echo=TRUE, eval=TRUE>>=
pheno_zeisel <- Biobase::pData(MouseZeiselBrain)
head(pheno_zeisel[,c("tissue","sex","age","level1_class","level2_class")])
@ %def


Gene names are stored in `featureData` slot:

<<zeisel_features, echo=TRUE, eval=TRUE>>=
genes_zeisel <- Biobase::featureNames(MouseZeiselBrain)
head(genes_zeisel)
@ %def


\begin{thebibliography}{1}

\bibitem{Jaitin2014}
Jaitin DA,  Kenigsberg E et al.
\newblock Massively Parallel Single-Cell RNA-Seq for Marker-Free Decomposition
of Tissues into Cell Types.
\newblock {\textit{Science}}. 343 (6172) 776-779, 2014. DOI: 10.1126/science.1247651

\bibitem{Deng2014}
Deng Q,  Ramskold D,  Reinius B,  Sandberg R.
\newblock Single-Cell RNA-Seq Reveals Dynamic, Random Monoallelic Gene
Expression in Mammalian Cells.
\newblock {\textit{Science}}.  343 (6167) 193-196, 2014. DOI: 10.1126/science.1245316

\bibitem{Zeisel2015}
Zeisel A, Munoz-Manchado AB, Codeluppi S \textit{et al}.
\newblock Cell types in the mouse cortex and hippocampus revleaed by
single-cell RNA-seq.
\newblock {\textit{Science}}.  34: 6226, 1138-1142, 2015. DOI:10.1126/science.aaa1934

\end{thebibliography}

\end{document}
